src/tribol/mesh/MfemData.cpp

```diff
@@ -504,8 +504,16 @@ void MfemMeshData::SetMaterialModulus( mfem::Coefficient& modulus_field )
 MfemMeshData::UpdateData::UpdateData( mfem::ParSubMesh& submesh, mfem::ParMesh* lor_mesh,
                                       const mfem::ParFiniteElementSpace& parent_fes,
                                       mfem::ParGridFunction& submesh_gridfn, SubmeshLORTransfer* submesh_lor_xfer,
-                                      const std::set<int>& attributes_1, const std::set<int>& attributes_2 )
-    : redecomp_mesh_{ lor_mesh ? redecomp::RedecompMesh( *lor_mesh ) : redecomp::RedecompMesh( submesh ) },
+                                      const std::set<int>& attributes_1, const std::set<int>& attributes_2,
+                                      RealT binning_proximity_scale )
+    : redecomp_mesh_{ lor_mesh ? redecomp::RedecompMesh(
```

Comment:
Make sure you have a comment somewhere about what the binning scale * max element size is for. I'm assuming this is for creating a max bounding box dimension for RCB or something like that?